docs/user-guide/advanced-usage.md

```diff
@@ -60,7 +60,7 @@ tamr.log_entry = log_entry
 ```
 
 
-## Custom HTTP requests and Unversioned API Access
+## Raw HTTP requests and Unversioned API Access
```

Comment:
@JuliaMalkin @nbateshaus we could remove the `Unversioned API Access` part of this header since we already cover unversioned API access in the FAQ and link to here already. Just a small simplification :smile: